tgui/packages/tgui/interfaces/CharacterPreferences/SavesTab.tsx

```diff
@@ -0,0 +1,51 @@
+import { Fragment } from 'inferno';
+import { useBackend } from '../../backend';
+import { Box, Button, Divider, LabeledList, Section } from '../../components';
+import { CharacterPreferencesData } from './type';
+
+export const SavesTab = (_props, context) => {
+  const { act, data } = useBackend<CharacterPreferencesData>(context);
+
+  return (
+    <Section title="Cloud Saves">
+      {data.cloudSaves ? (
+        <Fragment>
+          {data.cloudSaves.map((save, index) => (
+            <Fragment key={save.name}>
+              <Cloudsave name={save.name} index={index} />
+              <Divider />
+            </Fragment>
+          ))}
+          <Box mt="5px">
+            <Button onClick={() => act('cloud-new')}>Create new save</Button>
+          </Box>
```

Comment:
Future change: disable the button if they're at their limit (and give a tooltip explaining why).